www/src/frameworks.ts

```diff
@@ -28,6 +28,7 @@ export const frameworks: {[framework: string]: Framework} = {
       monthFrom: 1,
       yearTo: 2021,
       monthTo: 12,
+      habitatid: null,
```

Comment:
In typescript we tend to use undefined instead of null to show empty values (I think there's some subtle difference but it's basically to try and make it consistent with the rest of the stuff). But actually thinking about it, maybe we should change this to an array to allow for future selection of more than one id? So empty array if no habitat id is given.

Actually thinking about it more, if we actually implemented multiple choice filtering in the frontend then it would work as a workaround for the habmos hierarchy issue. Just a thought anyway, you might want to check back with Paul.

Reply:
Thanks Cathy.  That's a great idea, I will raise that as a final suggestion to Paul and Lisa.  But if it's OK I will just change this property to 'undefined' for now,  because to support multiselect we would also have to change the back end lambda which has already been deployed.  If we go ahead with the idea this will become an empty array